### ISSUE TITLE
Panel freezes when using pages field (Chrome)

### ISSUE THREAD

**Issue by sashtown** [getkirby/kirby#2290]
**Describe the bug**  
When trying to select a page item via the pages field in Chrome the Panel freezes instantly right after clicking on the field. I need to close the window to get back to the Panel again. (It works in FF though.)

**To Reproduce**
1. Go to the Panel.
2. Click on a pages field to select an item.

**Expected behavior**  
Ability to select a page item from the list.

**Kirby Version**  
3.3.0

**Console output**  
```
Uncaught RangeError: Maximum call stack size exceeded.
    at a.focus (app.js:1)
    at HTMLBodyElement.<anonymous> (app.js:1)
    at a.onClose (app.js:1)
    at oe (vendor.js:41)
    at a.n (vendor.js:41)
    at oe (vendor.js:41)
    at a.t.$emit (vendor.js:41)
    at a.close (app.js:1)
    at a.close (app.js:1)
    at a.blur (app.js:1)
```

**Desktop**  
 - OS: MacOS 10.14.6
 - Browser: Chrome 
 - Version: 78.0.3904.70

**Comment by afbora** [comment]
@sashtown I can't reproduce this issue on 3.3.0/Chrome/Windows. Such errors can often be caused by a cache or other plugins.

![pages-field](https://user-images.githubusercontent.com/3393422/68377018-b561ee80-015a-11ea-91e9-421a24a64a03.gif)

**Comment by sashtown** [comment]
@afbora Jap, thought about that, too. But even after deleting cache and removing the plugins I used the error remains. And again: It works in FF.

**Comment by afbora** [comment]
What about try to on starterkit? 

And could you replace beautified `app.js` (media/panel/xxx/js/app.js) with that to get more details on debugging and share us the console output:  [app.js.zip](https://github.com/getkirby/kirby/files/3818963/app.js.zip)

**Comment by sashtown** [comment]
It works with the starterkit. This is pretty weird... The console output is the same: Countless blocks of the output I posted above.  

The thing I don't get is: It works in FF and if I go back to Kirby 3.2.5 it also works again in Chrome. 🤔

**Comment by sashtown** [comment]
Ok, there seems to be a problem when adding a multiselect field in the same blueprint. If I drop the multiselect the pages field works fine. Just tested it with the Starterkit. 

Could you please try to reproduce that on your side with something like:

```yaml

mypages:
  label: My pages
  type: pages

categories:
  label: Categories
  type: multiselect
  options:
    design: Design
    architecture: Architecture
    photography: Photography
```

**Comment by afbora** [comment]
A really weird bug! I can't close the page now 😱

**Comment by sashtown** [comment]
Yes, that's weird. And the fan of my machine was running like crazy. 😂 
But at least you could reproduce it on your side as well.

**Comment by afbora** [comment]
It enters to vicious loop between these two methods:

**1. this.focus();**
https://github.com/getkirby/kirby/blob/develop/panel/src/components/Dialogs/Dialog.vue#L93

**2. autofocus.focus();**
https://github.com/getkirby/kirby/blob/develop/panel/src/components/Dialogs/Dialog.vue#L131

**Comment by afbora** [comment]
This issue about this PR #2137 and following line:

https://github.com/getkirby/kirby/blob/develop/panel/src/components/Forms/Field/MultiselectField.vue#L56

**Comment by olach** [comment]
I can confirm this issue. A client told me today they couldn't publish new articles. I had to downgrade to 3.2.5.

For us, the issue occurs when creating a new page/draft. The new page dialog is unresponsive  with the error message `Uncaught RangeError: Maximum call stack size exceeded.` in dev tools. See screen shot.

![multiselect error](https://user-images.githubusercontent.com/1300644/68939874-064c9500-07a2-11ea-84a1-eedc68337b09.png)

On this page we have a multiselect field. If I remove the field, the error is gone.

The error occurs in Safari and Chrome, not in Firefox.

**Comment by afbora** [comment]
@olach Thank you for confirmation. It will soon be fixed in version 3.3.1.

**Comment by bastianallgeier** [comment]
✅
